src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,114 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
```

Comment:
The class should not start with an empty line. Please remove any unnecessary empty lines at the beginning of class or method implementations.